src/models/client.ts

```diff
@@ -0,0 +1,11 @@
+export interface Client {
+  mac_address: string
+  alias: string
+  event_id: string
+  last_heartbeat: number
+  status: boolean
+}
+
+export interface Heartbeat {
+  mac_address: string
+}
```

Comment:
Identical to RPI and Heartbeat interfaces from `/models/checkin`. Could we combine into one file? No is an acceptable answer if it is accompanied with a valid argument.

Reply:
We should remove the interfaces from that file.

Reply:
Fixed in most recent commit.